supabase/migrations/20230606180645_add_bookmarked_projects.sql

```diff
@@ -0,0 +1,24 @@
+create table public.user_bookmarks (
+    "project" text not null,
```

Comment:
could this be a foreign key to the `id` of the `projects` table?

```suggestion
  project_id bigint not null references public.projects(id),
```